content/blog/2023/01/release-1_5_0.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Kubewarden 1.5.0 release
+authors:
+- José Guilherme Vanz
+date: 2023-01-18
+---
+
+Today we're pleased to announce the availability of Kubewarden 1.5.0!
+
+This release brings the usual amount of small bug fixes, dependency updates and a major security feature.
+Let's take a closer look!
+
+## Policy evaluation timeout
+
+The Kubewarden team is constantly working to improve the security posture of the project. As part of this efforts,
+we're excited to introduce the new "policy evaluation timeout" feature.
+
+Starting from this release, Policy Server will interrupt the evaluation of admission requests after a certain amount of time has elapsed.
```

Comment:
This needs to be phrased better IMO. Certain amount of time has elapsed after what? I understand it prevents the Policy Server from DOS attacks by preventing it from infinite loops and that we explain this context in the next sentence. But we need to make it clear. I'd recommend we set the context first by stating the problem first a.k.a. talk about potential DOS attacks or running out of compute resources on account of infinite loops. Then present the solution i.e. the Policy Server interrupting the evaluation of admission requests